### PR TITLE
Project-Movies Elle Frankenberg Anna Lenksjö

### PR DESCRIPTION
**Pull request by ElleFrankenberg**
Link to my deployed project:
https://suspicious-wright-21d05f.netlify.app
My reflections on how this week's project turned out:
It was a fun project we wished we had more time to do the red level requirements.
Things I'd like to have clarified or explained in more detail:
More about catch errors.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+  <BrowserRouter>
+    <Switch>
+      <Route path ="/" exact>
+        <MovieList />
+      </Route>
+      <Route path="/movies/:movieId" exact>
+        < MovieDetails/>
+      </Route>
+      < Route>
+        <NotFound />
```
Comment (TobiasHagberg) [inline code/src/App.js:20]: Really nice thought of implementing this component but we can't really see how it's used in the project? We do however see your comment in readme and that you might have had an issue with this. 
Nevertheless good job! :)

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,150 @@
...
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*********MOVIE LIST***********/
+
+.movies-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (TobiasHagberg) [inline code/src/index.css:24]: We applaud you for using grid, hopefully we will all get the opportunity to use even more and perhaps with some more explanation and use cases.

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,150 @@
...
+  html {
+    font-size: 1.2px;
+  }
+}
+
+@media (min-width: 992px) {
+  html {
+    font-size: 1.5px;
+  }
+}
```
Comment (TobiasHagberg) [inline code/src/index.css:148]: We don't really follow how you have used the concept of the rem units. To us it's almost as you have turned it all around but then again we are happy to be missproven :) From what we se and understand it must have been really hard to come up with the correct font size?! :)

File: code/src/pages/NotFound.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const NotFound = () => {
```
Comment (TobiasHagberg) [inline code/src/pages/NotFound.js:4]: Again, nice and professional touch but we can't really provoke the site to get this to show. Netlify's own error message seems to take over!

**Review by TobiasHagberg — APPROVED** [review]
Hi Elle and Anna,
All and all this is a really good project that meets all the requirements and are according to our standards approved!👏

As our comments will suggest we can't really se the Not Found component and maybe this is what you mention in the readme? The use of rem is for us somewhat confusing but then again we are no experts and you might have comments and explanations that will learn us something😄

Nice job and good luck to the both of you with this weeks project! Go Tigers!!!

File: code/src/pages/MovieDetails.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  const { movieId } = useParams()
+  const [movie, setMovie] = useState() 
+
+  const MOVIE_DETAILS_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=e20be4c51fce8750763fdb57c2e46728&language=en-US](https://api.themoviedb.org/3/movie/%7Bmovie_id%7D?api_key=%7Bapi_key%7D&language=en-US)`
+
+useEffect(() => {
+  fetch(MOVIE_DETAILS_URL)
+  .then (response => response.json())
+  .then (json => setMovie(json))
+  .catch(err => console.error(err))
```
Comment (KarolCislowski) [inline code/src/pages/MovieDetails.js:16]: Please remember to remove all console outputs from your code before submitting. 👮🏻

File: code/src/pages/NotFound.js (hunk 4 above)
Comment (KarolCislowski) [reply code/src/pages/NotFound.js:4]: if you are using netlify to host your app: create inside the `public` folder the `_redirects` file and save this line in it: `/* /index.html 200`

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Nice project but please remove all console outputs.

File: code/src/pages/MovieDetails.js (hunk 5 above)
Comment (ElleFrankenberg) [reply code/src/pages/MovieDetails.js]: Hi! we have now removed the console output 🐧

**Review by KarolCislowski — APPROVED** [review]

**Pull request closed without merging**